packages/devtools_app/lib/src/ui/vm_flag_widgets.dart

```diff
@@ -68,9 +70,10 @@ class ProfileGranularityDropdown extends StatelessWidget {
           style: Theme.of(context).textTheme.bodyText2,
           value: safeValue,
           items: [
-            _buildMenuItem(ProfileGranularity.low),
-            _buildMenuItem(ProfileGranularity.medium),
-            _buildMenuItem(ProfileGranularity.high),
+            _buildMenuItem(ProfileGranularity.low) as DropdownMenuItem<String>,
```

Comment:
similar comment. Make the return type of _buildMenuItem be `DropdownMenuItem<String>`. Then we should be able to remove the casts

Reply:
done